spec/fixtures/responses/queries/freaks/freaks.json

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
Where is this used from?

Reply:
I think this file was from rebase. Now it's gone